README.md

```diff
@@ -77,13 +77,49 @@ You can also disable the conversion of the built-in components `{{link-to}}`, `{
 }
 ```
 
+You can also disable the final **Prettier** formatting by adding `skipPrettier` as follows:
+
+**config/anglebrackets-codemod-config.json**
+
+```js
+{
+  "helpers": [],
+  "skipPrettier": true
```

Comment:
Perhaps, instead of a boolean ,it would be better to add a `formatter` option whose possible values can be `prettier` (the default) and `none`? That would allow us the option of adding a new formatter (eg. `ember-template-recast` or `prettier-experimental`) in future

Reply:
sounds good.  I'll implement that.